### PR TITLE
[PHP 8.4] sockets constant list update

### PR DESCRIPTION
**Pull request by devnexen**

### CODE REVIEW

File: reference/sockets/constants.xml [hunk 1]
```diff
@@ -103,6 +103,54 @@
...
+  </varlistentry>
+  <varlistentry xml:id="constant.sock-dccp">
+   <term>
+    <constant>SOCK_DCCP</constant>
+    (<type>int</type>)
+   </term>
+   <listitem>
+    <simpara>
+     Set the socket to a datagram congestion control protocol.
+     Available as of PHP 8.4.0 (NetBSD only)
```
Comment (Girgias) [inline reference/sockets/constants.xml:126]: Please add a full stop at the end of the "Available as of PHP 8.4.0." for all the constants
```suggestion
     Available as of PHP 8.4.0. (NetBSD only)
```

File: reference/sockets/constants.xml [hunk 2]
```diff
@@ -2760,6 +2808,54 @@
...
   </varlistentry>
+  <varlistentry xml:id="constant.ip-portrange">
+   <term>
+    <constant>IP_PORTRANGE</constant>
+    (<type>int</type>)
+   </term>
+   <listitem>
+    <simpara>
+     Set the port	range used for selecting a local port	number.
+     Available as of PHP 8.4.0 (FreeBSD/NetBSD/OpenBSD only)
```
Comment (Girgias) [inline reference/sockets/constants.xml:2819]: Tabs usage?
```suggestion
     Set the port range used for selecting a local port number.
     Available as of PHP 8.4.0 (FreeBSD/NetBSD/OpenBSD only)
```

File: reference/sockets/constants.xml [hunk 3]
```diff
@@ -2760,6 +2808,54 @@
...
+  </varlistentry>
+  <varlistentry xml:id="constant.ip-portrange-default">
+   <term>
+    <constant>IP_PORTRANGE_DEFAULT</constant>
+    (<type>int</type>)
+   </term>
+   <listitem>
+    <simpara>
+     Use  the default	range of port values.
+     Available as of PHP 8.4.0 (FreeBSD/NetBSD/OpenBSD only)
```
Comment (Girgias) [inline reference/sockets/constants.xml:2831]: Ditto tabs
```suggestion
     Use the default range of port values.
     Available as of PHP 8.4.0 (FreeBSD/NetBSD/OpenBSD only)
```

File: reference/sockets/constants.xml [hunk 4]
```diff
@@ -2760,6 +2808,54 @@
...
+  </varlistentry>
+  <varlistentry xml:id="constant.ip-portrange-low">
+   <term>
+    <constant>IP_PORTRANGE_LOW</constant>
+    (<type>int</type>)
+   </term>
+   <listitem>
+    <simpara>
+     Use a low	range of port values.
+     Available as of PHP 8.4.0 (FreeBSD/NetBSD/OpenBSD only)
```
Comment (Girgias) [inline reference/sockets/constants.xml:2855]:
```suggestion
     Use a low range of port values.
     Available as of PHP 8.4.0 (FreeBSD/NetBSD/OpenBSD only)
```

**Review by Girgias — CHANGES_REQUESTED** [review]
Please double-check that there are no tab characters used

**Comment by Girgias** [comment]
Why was this closed?

**Comment by devnexen** [comment]
I think that was an accident "wrong tab" thing ...

**Comment by devnexen** [comment]
Reopening

**Pull request closed without merging**